src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs

```diff
@@ -22,6 +23,15 @@ internal sealed class SqlActivitySourceHelper
     public static readonly ActivitySource ActivitySource = new(ActivitySourceName, Assembly.GetPackageVersion());
     public static readonly string ActivityName = ActivitySourceName + ".Execute";
 
+    public static readonly string MeterName = AssemblyName.Name!;
+    public static readonly string MeterVersion = AssemblyName.Version!.ToString();
```

Comment:
The `ActivitySource` is using `Assembly.GetPackageVersion()`. Should probably use that to make these consistent.

This whole section would probably be better written something like

```csharp
    public static readonly Assembly Assembly = typeof(SqlActivitySourceHelper).Assembly;
    public static readonly AssemblyName AssemblyName = Assembly.GetName();
    public static readonly string InstrumentationScopeName = AssemblyName.Name!;
    public static readonly string InstrumentationScopeVersion = Assembly.GetPackageVersion();

    public static readonly ActivitySource ActivitySource = new(InstrumentationScopeName, InstrumentationScopeVersion);
    public static readonly Meter Meter = new(InstrumentationScopeName, InstrumentationScopeVersion);

```